overlays/46-audio-receiver/usr/local/bin/a2dp.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export PULSE_RUNTIME_PATH="/run/user/$(id -u neon)/pulse/"
```

Comment:
Could this use `$USER` instead of `neon` to support arbitrary usernames? or maybe source the environment file suggested above for systemd services

Reply:
Switching to environment file, to limit the number of places a replacement might need to be made.